app/controllers/users_controller.rb

```diff
@@ -2,5 +2,22 @@ class UsersController < ApplicationController
   def show
     @user = User.find(params[:id])
     authorize @user
+    @user[:host_avg_rating] = user_avg_rating(@user.id)
+    @user.save
   end
+
+   def user_avg_rating(user_id)
```

Comment:
Again the same method? How many times do we need to repeat the same code? DRY! 😛